graphblas/core/ss/unary.py

```diff
@@ -0,0 +1,63 @@
+from ... import backend
+from ...dtypes import lookup_dtype
+from ...exceptions import check_status_carg
+from .. import NULL, ffi, lib
+from ..operator.base import TypedOpBase
+from ..operator.unary import TypedUserUnaryOp, UnaryOp
+from . import _IS_SSGB7
+
+ffi_new = ffi.new
+if _IS_SSGB7:
+    # JIT was introduced in SuiteSparse:GraphBLAS 8.0
+    import suitesparse_graphblas as ssgb
+
+    raise ImportError(
+        "JIT was added to SuiteSparse:GraphBLAS in version 8; "
+        f"current version is {ssgb.__version__}"
+    )
+
+
+class TypedJitUnaryOp(TypedOpBase):
+    __slots__ = "_jit_c_definition"
+    opclass = "UnaryOp"
+
+    def __init__(self, parent, name, type_, return_type, gb_obj, jit_c_definition):
+        super().__init__(parent, name, type_, return_type, gb_obj, name)
+        self._jit_c_definition = jit_c_definition
+
+    @property
+    def jit_c_definition(self):
+        return self._jit_c_definition
+
+    __call__ = TypedUserUnaryOp.__call__
+
+
+def register_new(name, jit_c_definition, input_type, ret_type):
```

Comment:
Similarly, should we get the name, input type, and return type from the C definition here?

Alternatively, should `name` not be required to match the name used in the C definition?

Should we call this function something else, such as `gb.unary.ss.register_jit`?

Also, should we consider adding `register_anonymous` or allowing it to be anonymous?